gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/medium_tests/gnomad_genomes_GRCh37_chrX_head2500_run_vep.json

```diff
@@ -12,6 +12,10 @@
     "shard_variants": "False",
     "num_workers": 2,
     "assertion_configs": [
+      {
+        "query": ["NUM_OUTPUT_TABLES"],
+        "expected_result": {"num_tables": 26}
+      },
```

Comment:
Could you please make sure all integration tests now have this check. And also for all of them the expected number must include `sample_info` table.

Reply:
Done.